src/orderbook/Order.py

```diff
@@ -0,0 +1,39 @@
+# Order to create an order object
+class Order:

```

Comment:
is that needed over regular dictionnary ? it is adding extra load for no reason

Reply:
Yeah true, we can do a dictionnary